modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/ProcessBuilderSource.testcase

```diff
@@ -678,8 +678,6 @@ definition {
 	@description = "Verify that the time format in Revision History tab is following the system language"
 	@priority = "4"
 	test TimeFormatFollowsSystemLanguageInRevision {
-		property portal.upstream = "quarantine";
-
```

Comment:
Hey @sirfabios,

You can use other commits when removing the quarantined property.